corehq/apps/data_analytics/malt_generator.py

```diff
@@ -40,8 +42,9 @@ def build_table(self):
             logger.info("Building MALT for {}".format(domain.name))
 
             for monthspan in self.monthspan_list:
-                for users in chunked(domain.all_users(), 10000):
-                    users_by_id = {user._id: user for user in users}
+                all_users = get_all_user_rows(domain.name, include_inactive=False, include_docs=True)
+                for users in chunked(all_users, 1000):
+                    users_by_id = {user['id']: CouchUser.wrap(user['doc']) for user in users}
```

Comment:
should this be calling `CouchUser.wrap_correctly()`?

Reply:
I believe the function it replaced did not, but it is probably more correct to do so. It only pulls `username`, `doc_type`, and `email` for the users so it should not matter. I will leave it up to you whether you think that change should block.

Reply:
fixed